### PR TITLE
Fix: Use right service and package name for archlinux

### PR DESCRIPTION
**Pull request by sebastianrakel**
#### Pull Request (PR) description
On Archlinux the package name for node_exporter and blackbox_exporter are prometheus-node-exporter and prometheus-blackbox-exporter

This PR fixes this

### CODE REVIEW

**Comment by dhoppe** [comment]
@sebastianrakel Thank you very much for your contribution, but you need to fix the Rspec tests:
* https://github.com/voxpupuli/puppet-prometheus/blob/master/spec/classes/blackbox_exporter_spec.rb#L33
* https://github.com/voxpupuli/puppet-prometheus/blob/master/spec/classes/node_exporter_spec.rb#L16

**Review by dhoppe — APPROVED** [review]

**Review by dhoppe — CHANGES_REQUESTED** [review]
Please fix the Rubocop, Travis CI related issues.

File: data/Archlinux.yaml [hunk 1]
```diff
@@ -3,3 +3,7 @@ prometheus::version: '2.2.0'
 prometheus::install_method: 'package'
 prometheus::bin_dir: '/usr/bin'
 prometheus::env_file_path: '/etc/default'
+prometheus::node_exporter::package_name: 'prometheus-node-exporter'
```
Comment (dhoppe) [inline data/Archlinux.yaml:6]: @sebastianrakel Why did you change the value of the parameter ```package_name```? This is part of the download URL and I do not think that ArchLinux uses different binaries. If you remove line 6 and 8, the tests should pass.

**Review by dhoppe — CHANGES_REQUESTED** [review]

File: data/Archlinux.yaml (hunk 1 above)
Comment (bastelfreak) [reply data/Archlinux.yaml:6]: Because archlinux provides native packages and that one is named `prometheus-node-exporter`.

Comment (dhoppe) [reply data/Archlinux.yaml:6]: @bastelfreak Ah. I did not know that. In that case we need an additional case for the tests like @sebastianrakel did for ```service_name```.

**Comment by bastelfreak** [comment]
I think this is obsolete now?
